docs/using-the-compiler.rst

```diff
@@ -159,9 +159,9 @@ Input Description
           [
             "bzzr://56ab...",
```

Comment:
Each of them is tried with the callback. Stops on first success.

Reply:
Ah nice! But it does not work with `file://`, does it?

Reply:
Actually, using the CLI it looks to me like the only think that will work will be plain old filesystem paths - everything is read through the read callback, so with soljson.js I guess it can be whatever type of URL supported by the respective callback, but the CLI callback just reads the filesystem, unless I'm missing something...